src/Layouts/HeaderBar.vala

```diff
@@ -62,35 +63,89 @@ public class Akira.Layouts.HeaderBar : Gtk.HeaderBar {
     construct {
         set_show_close_button (true);
 
-        var menu_items = new Gtk.Menu ();
+        var menu_popover_grid = new Gtk.Grid ();
+        menu_popover_grid.margin_bottom = 3;
+        menu_popover_grid.orientation = Gtk.Orientation.VERTICAL;
+        menu_popover_grid.width_request = 220;
 
-        var new_window = new Gtk.MenuItem.with_label (_("New Window"));
-        new_window.action_name = Akira.Services.ActionManager.ACTION_PREFIX + Akira.Services.ActionManager.ACTION_NEW_WINDOW;
-        menu_items.add (new_window);
-        menu_items.add (new Gtk.SeparatorMenuItem ());
+        var new_window_button = new Akira.Partials.PopoverButton (
+            _("Open New Window"), {"<Ctrl>n"});
```

Comment:
Is there a reason for putting those on a new line?

Reply:
I started formatting with a soft 80 character limits and a hard 100.
It looks a bit weird and I'll probably avoid it, tho.
What do you think?